tools/ci_build/op_registration_validator.py

```diff
@@ -47,7 +47,7 @@ def process_registration(
         key = domain + ":" + operator
         prev_start, prev_end = self.last_op_registrations[key] if key in self.last_op_registrations else (None, None)
 
-        if prev_start:
+        if prev_start and start_version > prev_start:
```

Comment:
I think this will skip validation of some kernel registrations. if we want to modify the registration validation, what we probably want to do is support registrations appearing in an arbitrary order.